packages/cf-component-select/src/Select.js

```diff
@@ -4,56 +4,21 @@ const ReactSelect = require('react-select');
 
 class Select extends React.Component {
   render() {
-    const SelectClass = this.props.async ? ReactSelect.Async : ReactSelect;
+    const { async, label, ...props } = this.props;
+    const SelectClass = async ? ReactSelect.Async : ReactSelect;
     return (
       <div className="cf-select">
-        {this.props.label && <label>{this.props.label}</label>}
+        {label && <label>{label}</label>}
         <SelectClass
-          // Simplify if this isnt a searchable select
-          searchable={this.props.searchable}
+          {...props}
           clearable={this.props.searchable}
           backspaceRemoves={this.props.searchable}
-          onChange={this.props.onChange}
-          onBlur={this.props.onBlur}
-          onFocus={this.props.onFocus}
-          loadOptions={this.props.loadOptions}
-          multi={this.props.multi}
-          value={this.props.value}
-          options={this.props.options}
-          placeholder={this.props.placeholder}
-          disabled={this.props.disabled}
-          cache={this.props.cache}
         />
       </div>
     );
   }
 }
 
-Select.propTypes = {
```

Comment:
[react-select already does this](https://github.com/JedWatson/react-select/blob/master/src/Select.js#L47)